tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -1729,3 +1729,58 @@ def test_max(is_module, xp, device, input, axis, dtype):
         return xp.max(a, axis)
     else:
         return a.max(axis)
+
+
+@chainer.testing.parameterize(*(
+    chainer.testing.product([
+        chainer.testing.from_pytest_parameterize(
+            'x1_shape,x2_shape', [
+                ((3, 2), (3, 2)),
+                ((), ()),
+                ((3, 2), (3, 1)),
+                ((2,), (3, 2)),
+            ]),
+        chainer.testing.from_pytest_parameterize(
+            'in_dtypes,out_dtype', _in_out_dtypes_arithmetic)
+    ])
+))
+class TestMinimum(op_utils.ChainerOpTest):
+
+    def setup(self, dtype):
+
+        x1_dtype, x2_dtype = self.in_dtypes
+
+        if (x1_dtype == 'float16' or x2_dtype == 'float16'):
+            self.check_backward_options.update({'rtol': 3e-3, 'atol': 3e-3})
+            self.check_double_backward_options.update(
+                {'rtol': 3e-3, 'atol': 3e-3})
+
+    def generate_inputs(self):
+        x1_shape = self.x1_shape
+        x2_shape = self.x2_shape
+        x1_dtype = self.x1_dtype
+        x2_dtype = self.x2_dtype
+        x1 = numpy.random.uniform(-1, 1, x1_shape).astype(x1_dtype)
```

Comment:
How about using `array_utils.uniform`? The lower bound and upper bound are not appropriate for non-float inputs.

Reply:
Do you suggest having a wider range to allow for more integers? (For example, -5, 5)

Reply:
That makes sense. Will make the change. Thanks for the suggestion.